content/sensu-enterprise/3.2/integrations/timescaledb.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "TimescaleDB"
+product: "Sensu Enterprise"
+version: "3.2"
+weight: 22
+menu:
+  sensu-enterprise-3.2:
+    parent: integrations
+---
+**ENTERPRISE: Built-in integrations are available for [Sensu Enterprise][1]
+users only.**
+
+- [Overview](#overview)
+- [Configuration](#configuration)
+  - [Example(s)](#examples)
+  - [Integration Specification](#integration-specification)
+    - [`timescaledb` attributes](#timescaledb-attributes)
+
+## Overview
+
+Send metrics to [TimescaleDB][2] using a PostgreSQL connection.
```

Comment:
> Do we need any further context around "using a PostgreSQL connection"?

@apaskulin How about something like this?

"TimescaleDB is an open-source time-series database powered by PostgreSQL. This integration allows Sensu Enterprise to send metrics directly to TimescaleDB using the PostgreSQL protocol."

Reply:
We can use https://www.postgresql.org/docs/current/static/protocol.html as a reference for PostgreSQL protocol if needed.